pkg/graphql/normalization_test.go

```diff
@@ -95,7 +95,41 @@ func TestRequest_Normalize(t *testing.T) {
 }`)
 	})
 
-	t.Run("should successfully normalize query and remove unused variables with no value provided", func(t *testing.T) {
+	t.Run("should successfully normalize query and remove unused variables and values", func(t *testing.T) {
+		const expectedVar = "query MySearch($s: String!){\n    search(name: $s){\n        ... on Human {\n            name\n        }\n    }\n}"
+		for _, v := range []Request{
+			{
+				OperationName: "MySearch",
+				Variables: stringify(map[string]interface{}{
+					"s":  "Luke",
+					"s2": nil,
```

Comment:
Are you sure that this will marshal to "s2": null in the final JSON?

Reply:
@jensneuse  I'm not sure, just a test case I wanted to cover! do you have a better idea?

Reply:
Instead of stringify, you could create another test where you hardcode "s2":null into the JSON string/bytes to make sure this case is covered.

Reply:
@jensneuse done https://github.com/TykTechnologies/graphql-go-tools/pull/141/commits/c208e7e7ed4fb1766a69157fc68e4933f31d5e40